draft-ietf-detnet-dp-sol-mpls-02.xml

```diff
@@ -1113,37 +1004,337 @@ Examples:
+     </t>
+     <t>
+       After an app flow is identified for a received DetNet MPLS
+       packet, as described above, an implementation MUST check if POF
+       is configured for that app flow.  When configured the
+       implementation MUST track the sequence number contained in
+       received d-CWs and MUST ensure that packets are processed in the
+       order indicated in the received d-CW sequence number field, which
+       may not be in the order the packets are received. As defined in
+       <xref target="dn-sn"/> the sequence number field length may be 16
+       or 28 bits, is incremened by one (1) for each new app flow packet
+       sent, and the field value can wrap.  The specific mechanisms used
+       for an implementation to identify the order of received packets
+       is an implementation choice.
+     </t>
+     <t>
+       Note that an implementation MAY wish to constrain the maximum
+       number of out of order packets that can be processed, on
+       platform-wide or per flow basis.  Some implementations MAY
+       support the provisioning of this number on either a platform-wide
+       or per flow basis.  The number of out of order packets that can
+       be processed also impacts the latency of a flow.
+     </t>
+   </section>
+  </section>
+  <section anchor="f-labels" title="F-Labels">
+     <t>
+       F-Labels are support the DetNet forwarding sub-layer. F-Labels
+       are used to provide LSP-based connectivity between DetNet service
+       sub-layer processing nodes.
+     </t>
+     <section anchor="f-labels-ssl"
+              title="Service Sub-Layer Processing Node Processing">
+       
+       <t>
+         DetNet MPLS end systems, edge nodes and relay nodes may operate
+         at the DetNet service sub-layer with understand of app-flows
+         and their requirements. As mentioned above, when operating at
+         this layer such nodes can push, pop or swap (pop then push)
+         S-Labels. In all cases, the F-Labels used for the app-flow are
+         always replaced and this section applies. 
+       </t>
+       <t>
+         When sending a DetNet flow, Zero or more F-Labels MAY be added
+         on top of an S-Label by the node pushing an S-Label. The
+         F-Labels to be pushed when sending a particular app-flow MUST
+         be provisioned per app-flow via configuration and the
+         controller plane, see <xref target="cp_considerations"/>.  To
+         allow for the omission of F-Labels, an implementation SHOULD
+         also allow an outgoing interface to be provisioned.
+     </t>
+     <t>
+       The Packet Replication Function (PRF) function MAY be supported
+       by an implementation for outgoing DetNet flows. When supported,
+       the same app-flow data will be sent over multiple outgoing
+       forwarding sub-layer LSPs.  To support PRF an implementation MUST
+       support the setting of different sets of F-Labels.  Hereto, to

```

Comment:
Compound flows may have different S-Label. So, add:
"different sets of F-Labels **and S-labels**"